canonical/vanilla-framework
# add side navigation accordion variant

## Done

- add side navigation accordion variant

Fixes https://github.com/canonical/vanilla-squad/issues/1380

## QA

- Open [demo](https://vanilla-framework-4550.demos.haus/docs/examples/patterns/side-navigation/accordion) 
- ensure the accordion works correctly on all screen sizes that we support

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

![Kapture 2022-08-25 at 10 56 52](https://user-images.githubusercontent.com/7452681/186621773-fbb9bf6b-7b9f-4969-9230-90528a5f50fd.gif)

<img width="900" alt="image" src="https://user-images.githubusercontent.com/7452681/186625859-3751f682-601f-4151-b0da-a61058a78550.png">
